### PR TITLE
Monorepo?

### PR DESCRIPTION
**Pull request by samchungy**
image

image

🤔

### CODE REVIEW

File: packages/eslint-config-skuba/package.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    "@typescript-eslint/eslint-plugin": "^6.13.2",
+    "@typescript-eslint/parser": "^6.13.2",
+    "eslint-config-seek": "^12.0.1",
+    "eslint-plugin-jest": "^27.6.0",
+    "eslint-plugin-tsdoc": "^0.2.14",
+    "eslint-plugin-yml": "^1.5.0"
+  },
+  "devDependencies": {
+    "eslint": "^8.11.0",
+    "skuba": "workspace:*",
```
Comment (72636c) [inline packages/eslint-config-skuba/package.json:38]: I think we may be able to remove this as a dev dep as `format` and `lint` can be run at the root: https://github.com/seek-oss/aws-codedeploy-hooks/blob/c493faa8265345302359cb5bf3aae8770f882b1f/package.json#L8-L9

File: pnpm-workspace.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+packages:
+  # https://github.com/changesets/changesets/issues/1133#issuecomment-1594844204
+  - .
```
Comment (72636c) [inline pnpm-workspace.yaml:3]: This is really neat, didn't realise we could accomplish this without displacing all the existing `skuba` code!

File: packages/eslint-config-skuba/package.json
Comment (72636c) [inline packages/eslint-config-skuba/package.json:1]: Do we want provenance? 😃

File: packages/eslint-config-skuba/package.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+  "version": "3.1.0",
+  "private": false,
+  "description": "ESLint config for skuba",
+  "homepage": "https://github.com/seek-oss/eslint-config-skuba#readme",
+  "bugs": {
+    "url": "https://github.com/seek-oss/eslint-config-skuba/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/seek-oss/eslint-config-skuba.git"
```
Comment (72636c) [inline packages/eslint-config-skuba/package.json:12]:
```suggestion
  "homepage": "https://github.com/seek-oss/skuba/tree/master/packages/eslint-config-skuba#readme",
  "bugs": {
    "url": "https://github.com/seek-oss/skuba/issues"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/seek-oss/skuba.git"
```

**Review by 72636c — APPROVED** [review]
Amazing, thank you!

**Review by AaronMoat — APPROVED** [review]
Would we want skuba-dive in here too?

**Comment by samchungy** [comment]
> Would we want skuba-dive in here too?

Ooooo mebe. I'll keep this PR on the eslint-config for now

**Pull request merged**